packages/codec/lib/wrap/priority.ts

```diff
@@ -10,6 +10,7 @@ import { maxValue, minValue, places } from "./utils";
 export function isMoreSpecificMultiple(
   types1: Format.Types.OptionallyNamedType[],
   types2: Format.Types.OptionallyNamedType[],
+  strictBooleans: boolean,
```

Comment:
I think this should be in an `options` style argument

Reply:
It could be if you really want, but this is sufficiently internal that I'm not really worried about that sort of thing -- there isn't a need to maintain a stable interface here, we can just change it as needed.  Adding yet another `options` here seems more confusing than just adding a flag.

Reply:
I really think we should always lean in favor of options-style arguments, but I won't press the issue here